config/domains.txt

```diff
@@ -20574,3 +20574,93 @@ uintacounty.com
 wrightwyoming.com
 wyomingbusiness.org
 wyomingtourism.org
+
+// Brazil Federal Government
+eb.mil.br
+brasil.gov.br
+dpu.def.br
+planalto.gov.br
+fazenda.gov.br
+planejamento.gov.br
+cultura.gov.br
+presidencia.gov.br
+mpf.mp.br
+mc.gov.br
+mec.gov.br
+mma.gov.br
+mre.gov.br
+saude.gov.br
+desenvolvimento.gov.br
+justica.gov.br
+mte.gov.br
+mme.gov.br
+defesa.gov.br
+mct.gov.br
+senado.gov.br
+camara.gov.br
+
+// Brazil Federal Institutes
+anatel.gov.br
+regulacao.gov.br
+anac.gov.br
+abin.gov.br
+unb.br
+ufac.br
```

Comment:
For all `u*.br`, these looks like universities. Even if federally funded, would it be accurate to say any user with e.g., a `ufac.br` email address is a government employee?

Reply:
Hey @benbalter I'm going to remove those to duplicate. In Brazil the federal University Employee's are actually Public Federal employee's.

In Brazil works in this way, the federal universities are part of the federal government.

Reply:
Actually, the onliest subdomain from those domains that may be used as non employee, could be the students subdomain, like `aluno.unb.br`, so I don't know how do a reverse rule.